addons/delivery_mondialrelay/static/src/components/mondialrelay_field.js

```diff
@@ -36,6 +36,7 @@ export class MondialRelayField extends Component {
                 if (!el) {
                     return;
                 }
+                // jquery is deprecated. Is it not possible to add this directly in the template?
```

Comment:
The mondialrelay widget thingy is jquery only unfortunately (it's a lib)

Reply:
We should probably do some lazy loading to avoid dependency.